### DISCUSSION TITLE
Log in to Plex failed: (401) unauthorized;

### DISCUSSION THREAD

**Discussion by igornovitzky** [Taxel/PlexTraktSync#896] [Support]
Hi
I have installed plextraktsync as plex but cannot authenticate to plex. I'm on a raspberry Pi 4B
I have used my correct plex credentials. Also, I have no PIN or 2FA for plex. I've tried both my username and the linked email address (I first created my account for plex 3y ago on via gmail so am entering those credentials).
I have linked my plex to multiple softwarrs without a problem - would appreciate any help with this issue. Thanks

Log in to Plex failed: (401) unauthorized; https://plex.tv/users/sign_in.xml <?xml version="1.0" encoding="UTF-8"?> <errors>   <error>You appear to be having trouble signing in to an account. You may wish to try resetting your password at https://plex.tv/reset</error> </errors> , Try again.

**Comment by glensc** [comment]
does your username contain some non-usual characters, like a space or comma or the like?

and to help understand your process you should post the output of the login session you had in the terminal.

**Comment by igornovitzky** [comment]
My email has normal characters; so does my username.
Thanks for replying.
Here it is with my email:

pi@raspberrypi:~ $ plextraktsync
INFO     PlexTraktSync [0.19.1]                                                 
Please enter your Plex username or e-mail: igor.novitzky@gmail.com
If you have 2 Factor Authentication enabled on Plex you can append the code to your password below (eg. passwordCODE)
Please enter your Plex password: 
Log in to Plex failed: (401) unauthorized; https://plex.tv/users/sign_in.xml <?xml version="1.0" encoding="UTF-8"?> <errors>   <error>Invalid email, username, or password.</error> </errors> , Try again.
Please enter your Plex username or e-mail [igor.novitzky@mgail.com]:

**Reply by glensc** [reply]
`@mgail.com` vs `@gmail.com` is that intentional? you could try logging in with username rather email, maybe better success.

**Reply by igornovitzky** [reply]
Yes, sorry, that's fixed.
Login with username:

pi@raspberrypi:~ $ plextraktsync
INFO     PlexTraktSync [0.19.1]                                                 
Please enter your Plex username or e-mail: mikhail2010
If you have 2 Factor Authentication enabled on Plex you can append the code to your password below (eg. passwordCODE)
Please enter your Plex password: 
Log in to Plex failed: (401) unauthorized; https://plex.tv/users/sign_in.xml <?xml version="1.0" encoding="UTF-8"?> <errors>   <error>Invalid email, username, or password.</error> </errors> , Try again.
Please enter your Plex username or e-mail [mikhail2010]:

**Comment by igornovitzky** [comment]
When I type in https://plex.tv/users/sign_in.xml, this is what I see

**Comment by igornovitzky** [comment]
![plexsignin](https://user-images.githubusercontent.com/104538583/165824494-2372ed02-0388-4cdf-9201-356f06c542b6.png)

**Comment by glensc** [comment]
Dunno then, try plex forums, this doesn't seem to be PlexTraktSync fault.

and if you found the solution, don't forget to update here too what was it, so other users can benefit too.

**Comment by glensc** [comment]
this
